tests/integration-tests/tests/storage/test_efs/test_multiple_efs/pcluster.config.yaml

```diff
@@ -29,22 +29,13 @@ Scheduling:
 # This compute subnet would be in a different AZ than head node for regions defined in AVAILABILITY_ZONE_OVERRIDES
 # See conftest for details
 SharedStorage:
-  - MountDir: {{ existing_efs_mount_dirs[0] }}
-    Name: efs1
-    StorageType: Efs
-    EfsSettings:
-      FileSystemId: {{ existing_efss[0] }}
   - MountDir: {{ new_efs_mount_dirs[0] }}
-    Name: efs2
+    Name: efs1
```

Comment:
As a part of this task could you please extend these tests to use different settings for the EFS (encrypted, performanceMode, ProvisionedThrougput, ThrougputMode, etc)?
I saw in all our EFS tests we're always testing default parameters.